lib/commands/project.ts

```diff
@@ -28,10 +31,125 @@ export class ProjectCommandBase implements ICommand {
+		this.indexHtml = path.join(this.projectDir, "index.html");
+	}
+
+	public execute(args: string[]): IFuture<void> {
+		return null;
+	}
+
+	allowedParameters: ICommandParameter[] = [];
+	protected isWebsiteProject(): IFuture<boolean> {
+		return ((): boolean => {
+			var result = true;
+
+			if(!this.$fs.exists(this.indexHtml).wait()) {
+				this.$logger.trace("Missing %s file, so this is not a mobile website project.", this.indexHtml);
+				result = false;
+			}
+
+			// if there's at least one cordova file, we can't say this is a website project
+			Object.keys(MobileHelper.platformCapabilities).forEach((platform) => {
+				var cordovaJsFileName = path.join(this.projectDir, util.format("cordova.%s.js", platform).toLowerCase());
+				if(this.$fs.exists(cordovaJsFileName).wait()) {
+					this.$logger.trace("Found %s file, so this project is not a mobile website project.", cordovaJsFileName);
+					result = false;
+				}
+			});
+
+			if(this.$fs.exists(this.tnsModulesDir).wait()) {
+				this.$logger.trace("Found %s directory, so this project is not a mobile website project.", this.tnsModulesDir);
+				result = false;
+			}
+
+			if(this.$fs.exists(this.bootstrapFile)) {
+				this.$logger.trace("Found %s file, so this project is not a mobile website project.", this.bootstrapFile);
+				result = false;
+			}
+
+			return result;
+		}).future<boolean>()();
+	}
+
+	protected isNativeProject(): IFuture<boolean> {
+		return ((): boolean => {
+			var result = true;
+
+			if(!this.$fs.exists(this.tnsModulesDir).wait()) {
+				this.$logger.trace("Missing %s directory, so this is not a native project.", this.tnsModulesDir);
+				result = false;
+			}
+
+			if(!this.$fs.exists(this.bootstrapFile).wait()) {
+				this.$logger.trace("Missing %s file, so this is not a native project.", this.bootstrapFile);
+				result = false;
+			}
+
+			if(this.$fs.exists(this.indexHtml).wait()) {
+				this.$logger.trace("Found %s file, so this is not a native project.", this.indexHtml);
+				result = false;
+			}
+
+			// if there's at least one cordova file, we can't say this is a native project
+			Object.keys(MobileHelper.platformCapabilities).forEach((platform) => {
+				var cordovaJsFileName = path.join(this.projectDir, util.format("cordova.%s.js", platform).toLowerCase());
+				if(this.$fs.exists(cordovaJsFileName).wait()) {
+					this.$logger.trace("Found %s file, so this is not a native project.", cordovaJsFileName);
+					result = false;
+				}
+			});
+
+			return result;
+		}).future<boolean>()();
+	}
+
+	protected isCordovaProject(): IFuture<boolean> {
+		return ((): boolean => {
+			var result = true;
+
+			Object.keys(MobileHelper.platformCapabilities).forEach((platform) => {
+				var cordovaJsFileName = path.join(this.projectDir, util.format("cordova.%s.js", platform).toLowerCase());
+				if(!this.$fs.exists(cordovaJsFileName).wait()) {
+					this.$logger.trace("Missing %s file, so this is not a hybrid project.", cordovaJsFileName);
```

Comment:
Missing %s. This is not a hybrid project.

(Maybe go with Apache Cordova project instead?)